Docs for dapr int/upgrade in k8s mode using GHCR and private registry (#2508)
dapr/docs: daprdocs/content/en/reference/cli/dapr-init.md

| `--slim`, `-s`        |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
| `--timeout`           |                      | `300`         | The wait timeout for the Kubernetes installation                                     |
| `--wait`              |                      | `false`       | Wait for Kubernetes initialization to complete                                       |

> |        N/A              |DAPR_DEFAULT_IMAGE_REGISTRY|          | It is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub as default, just unset this env variable.|

Reviewer: ```suggestion
|        N/A              |DAPR_DEFAULT_IMAGE_REGISTRY|          | It is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub, unset the environment variable or leave it blank|
```

```bash
dapr init -k --set global.tag=1.0.0 --set dapr_operator.logLevel=error
```

> You can also specify a private registry to pull container images from. As of now `dapr init -k` does not use specific images for sentry, operator, placement and sidecar. It relies on only Dapr runtime container image `dapr` for all these images.

Reviewer 2: ```suggestion
You can specify a private registry to pull container images from. 

Note: `dapr init -k` does not use the specific images for Sentry, Operator, Placement and sidecar. It relies on the Dapr runtime container image `dapr` for all these images.
```

Reviewer 2: I presume the "note" above is a bug that will be fixed and then this comment will be removed, yes?

Author: @msfussell  Thats true. There is already an issue created for this - https://github.com/dapr/dapr/issues/4712

Also, Could you please update label as "waiting for code pr to be merged", as the code PR is under review at the moment.
